src/__tests__/utilities/integration-test.utilities.js

```diff
@@ -0,0 +1,127 @@
+import { logError, logInfo } from "../../middleware/logging";
```

Comment:
On my machine, when I run all unit tests with `./tasks test_unit` this file `integration-test.utilities.js` is counted as a failed test suite. Can you check whether it is the same in your machine?

 FAIL  src/__tests__/utilities/integration-test.utilities.js
  ● Test suite failed to run

    Your test suite must contain at least one test.

Reply:
Addressed here https://github.com/nhsconnect/prm-repo-ehr-out-service/pull/24/commits/002157a9448236a322357b1568445b8554e4272d